rclpy/src/rclpy/_rclpy.c

```diff
@@ -25,7 +25,7 @@
 #include <rcl/remap.h>
 #include <rcl/time.h>
 #include <rcl/validate_topic_name.h>
-#include <rcl_interfaces/msg/parameter_type__struct.h>
+#include <rcl_interfaces/msg/detail/parameter_type__struct.h>
```

Comment:
As an alternative this could also just include `rcl_interfaces/msg/parameter_type.h`?

Reply:
I agree. I think the "detail" namespace should discourage users (including `rclpy`) from using those headers directly.

Reply:
Changed in cea1736f4ede0610b16298d85e0b8fa6125efeaa.